CustomFileFormats/DynamicInputTextures.md

```diff
@@ -0,0 +1,136 @@
+# Dolphin Dynamic Input Textures Specification (v1)
+
+## Format
+Dynamic Input Textures are generated textures based on a user's input formed from a group of png files and json files.
+
+```
+\__ Dolphin User Directory
+    \__ Load (Directory)
+        \__ DynamicInputTextures (Directory)
+            \__ GAMEID (Directory)
+                \__ PNG and JSON GO HERE
+```
+
+### PNG files
+
+At a minimum two images are required to support the generation and any number of 'button' images.  These need to be in PNG format.
+
+### JSON files
+
+You need at least a single json file that describes the generation parameters.  You may have multiple JSON files if you prefer that from an organizational standpoint.
+
+#### Possible fields in the JSON for a texture
+
+In each json, one or more generated textures can be specified.  Each of those textures can have the following fields:
+
+|Identifier               |Required | Since |
+|-------------------------|---------|-------|
+|``colorkey_image``       | **Yes** | v1    |
+|``fallback_image``       | **Yes** | v1    |
+|``emulated_controls``    | **Yes** | v1    |
+|``device``               | No      | v1    |
+|``host_controls``        | No      | v1    |
+
+*colorkey_image* - an image with colored boxes, will be used as a base image, so any non-input related sections can be upscaled/redrawn if desired
+
+*fallback_image* - after looking at the colorkey_image, if the color isn't mapped to a specific button, it will use this image as a reference for what to put in the color. This should be the same size/quality as the colorkey_image but without the colored boxes
```

Comment:
I personally prefer identifiers to be monospaced, and at a glance it seems `ResourcePacks.md` also does this.
Would you mind updating them here?
```suggestion
*fallback_image* - after looking at the ``colorkey_image``, if the color isn't mapped to a specific button, it will use this image as a reference for what to put in the color. This should be the same size/quality as the ``colorkey_image`` but without the colored boxes
```

Reply:
Did you mean instead of being italicized?  I had made one italicized to visually differentiate them from the table which is monospaced.  I can try monospacing everything though.  I wanted the descriptions in the table but they are much longer than the ones in ResourcePacks

Reply:
I mainly mean use of previously (emphasised or not) identifiers to set them apart from plain-text; whether thats italicized or monospaced or whatever is up to you (as long as it is consistent)

Reply:
Ok updated.  Please let me know if this isn't what you had in mind.